discovery-provider/src/api/v1/helpers.py

```diff
@@ -185,6 +185,7 @@ def to_dict(multi_dict):
 
 search_parser = reqparse.RequestParser()
 search_parser.add_argument('query', required=True)
+search_parser.add_argument('only_downloadable', required=False, default=False)
```

Comment:
let's double check w/ forrest on the nomenclature for this guy

Reply:
Already did! This is better than `downloadable` or something because that would imply that if downloadable is false, regular search wouldn't return downloadable tracks etc

Reply:
:)